app/views/home/_welcome.html.erb

```diff
@@ -62,13 +62,13 @@
       </ul>
 
       <p>
-        For more resources and training materials spanning the entire research data life cycle, see the <a href="https://alliancecan.ca/en/services/research-data-management/learning-and-training/training-resources/"> Portage Network Training Resources. </a>
+        For more resources and training materials spanning the entire research data life cycle, see the <a href="%{training_resources_path}"> Portage Network Training Resources. </a>
       </p>
 
       <p>
         The DMP Assistant was adapted from the <a href="https://dcc.ac.uk/"> Digital Curation Centre (DCC)</a>’s <a href="https://dmponline.dcc.ac.uk/">DMPonline</a> tool, and uses the DMP Roadmap codebase developed by DCC and the <a href="https://cdlib.org/services/uc3/">University of California Curation Center (UC3).</a>
       </p>
-    })
+    }),manage_data_path: how_to_manage_your_data_path, training_resources_path: training_resources_path)
```

Comment:
It is a very minor detail, we should keep consistency and change ```manage_data_path``` to ```how_to_manage_your_data_path``` on lines 71 and 35.

Reply:
Sounds good. Will fix that and then run the translation:sync command with prod key.